src/EA.Iws.DataAccess/Repositories/Reports/ExportMovementsRepository.cs

```diff
@@ -103,24 +103,36 @@ FROM[Notification].[MovementOperationReceipt] MOR
 
             var movementData = await context.Database.SqlQuery<ExportMovementsData>(query, parameters.ToArray()).SingleAsync();
 
-            var userActions = await context.Database.SqlQuery<UserActionData>(
-                @"SELECT
+            string userActionQuery = string.Format(@"SELECT
                     A.OriginalValue,
                     A.NewValue,
                     A.RecordId
                 FROM[Auditing].[AuditLog] AS A
                 INNER JOIN[Notification].[Movement] AS M ON M.Id = A.RecordId
                 INNER JOIN[Notification].[Notification] AS N ON M.NotificationId = N.Id
+                {0}
                 LEFT JOIN[Person].[InternalUser] AS IU ON A.UserId = IU.UserId
                 WHERE TableName = '[Notification].[Movement]'
                     AND EventType != 0
                     AND IU.UserId IS NULL
                     AND N.CompetentAuthority = @ca
                     AND A.EventDate BETWEEN @from AND @to
-                ORDER BY RecordId, EventDate",
+                    {1}
+                ORDER BY RecordId, EventDate", organisationJoin, organisationQuery);
+
+            List<SqlParameter> userActionParameters = new List<SqlParameter>
+            {
                 new SqlParameter("@ca", (int)competentAuthority),
                 new SqlParameter("@from", from),
-                new SqlParameter("@to", to)).ToListAsync();
+                new SqlParameter("@to", to)
+            };
+
+            if (organisationFilter != null)
+            {
+                userActionParameters.Add(new SqlParameter("@org", organisationName));
```

Comment:
Do we need this `@org` parameter? If I'm seeing this correct, we never make a reference of this parameter within the SQL query, unlike `@ca`, `@from` and `@to`. Also noticed that it's present at Line 101 too for the first SQL query.